drivers/bus/virtual_bus.c

```diff
@@ -44,7 +44,7 @@ static int virtbus_uevent(struct device *dev, struct kobj_uevent_env *env)
 {
 	struct virtbus_device *vdev = to_virtbus_dev(dev);
 
-	if (add_uevent_var(env, "MODALIAS=%s%s", VIRTBUS_MODULE_PREFIX,
+	if (add_uevent_var(env, "MODALIAS=%s%s", VIRTUALBUS_MODULE_PREFIX,
```

Comment:
if you rename on line 45 this should also be virtualbus_device = to_virtualbus_dev